projects/github.com/xiph/rav1e/package.yml

```diff
@@ -0,0 +1,27 @@
+distributable:
+   url: https://github.com/xiph/rav1e/archive/refs/tags/v{{ version }}.tar.gz
+   strip-components: 1
+
+versions:
+  github: xiph/rav1e/tags  # reads github tags from github
+  strip: /^(?!v).*/
+
+dependencies:
+  rust-lang.org: ">=1.60"
+  rust-lang.org/cargo: "*"
```

Comment:
It doesn't have run time dependency on rust or cargo.

Reply:
Fixed